base64ct/src/encoding.rs

```diff
@@ -296,16 +292,14 @@ fn is_pad_ct(input: u8) -> i16 {
 
 #[inline(always)]
 const fn encoded_len_inner(n: usize, padded: bool) -> Option<usize> {
-    // TODO: replace with `checked_mul` and `map` on stabilization
```

Comment:
1.47+ lets us use `checked_mul`, but unfortunately `map` is not `const fn` in any release to date